dashboard/src/components/AppView/AccessURLTable/AccessURLTable.v2.tsx

```diff
@@ -0,0 +1,184 @@
+      {URL}
+    </a>
+  ));
+}
+
+function getNotes(resource?: IResource) {
+  if (!resource) {
+    return <span>Unknown</span>;
+  }
+  const ips: Array<{ ip: string }> = get(resource, "status.loadBalancer.ingress", []);
+  if (ips.length) {
+    return <span>IP(s): {ips.map(ip => ip.ip).join(", ")}</span>;
+  }
+  return (
+    <span>
+      Not associated with any IP.{" "}
+      <Tooltip
+        label="pending-tooltip"
+        id={`${resource.metadata.name}-pending-tooltip`}
+        icon="help"
+        position="bottom-left"
+        large={true}
+        iconProps={{ solid: true, size: "sm" }}
+      >
+        Depending on your cloud provider of choice, it may take some time for an access URL to be
+        available for the application and the Service will stay in a "Pending" state until a URL is
+        assigned. If using Minikube, you will need to run <code>minikube tunnel</code> in your
+        terminal in order for an IP address to be assigned to your application.
+      </Tooltip>
+    </span>
+  );
+}
+
+export default function AccessURLTable({
+  services,
+  ingresses,
+  ingressRefs,
+  getResource,
+}: IAccessURLTableProps) {
+  useEffect(() => {
+    // Fetch all related Ingress resources. We don't need to fetch Services as
+    // they are expected to be watched by the ServiceTable.
+    ingressRefs.forEach(r => getResource(r));
+  }, [ingressRefs, getResource]);
+
+  if (isSomeResourceLoading(ingresses.concat(services))) {
+    return <LoadingWrapper loaded={false} />;
+  }
+  if (!hasItems(services, ingresses)) {
+    return null;
+  }
+
+  let result = <p>The current application does not expose a public URL.</p>;
+  const publicServices = filterPublicServices(services);
+  if (publicServices.length > 0 || ingresses.length > 0) {
+    const columns = [
+      {
+        accessor: "url",
+        Header: "URL",
+      },
+      {
+        accessor: "type",
+        Header: "Type",
+      },
+      {
+        accessor: "notes",
+        Header: "Notes",
+      },
+    ];
+    const allIngresses = flattenIngresses(ingresses);
+    const data = publicServices
+      .map(svc => {
+        const urlItem = GetURLItemFromService(svc.item);
+        return {
+          url: urlItem.isLink ? getAnchors(urlItem.URLs) : urlItem.URLs.join(","),
+          type: urlItem.type,
+          notes: svc.error ? <span>Error: {svc.error.message}</span> : getNotes(svc.item),
+        };
+      })
+      .concat(
+        allIngresses.map(ingress => {
+          return {
+            url: ingress.item ? getAnchors(GetURLItemFromIngress(ingress.item).URLs) : "Unkown",
+            type: "Ingress",
+            notes: ingress.error ? (
+              <span>Error: {ingress.error.message}</span>
+            ) : (
+              getNotes(ingress.item)
+            ),
+          };
+        }),
+      );
+    result = <Table data={data} columns={columns} />;
+  }
+  return (
+    <>
+      <h6>Access URLs</h6>
+      {result}
+    </>
```

Comment:
Is this part of another `section`? If not, please make this block a section itself.
```suggestion
    <section aria-labelledby="access-urls-title">
      <h6 id="access-urls-title">Access URLs</h6>
      {result}
    </section>
```